# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/VideoStreamFactory.cpp
# Commit: 1c3926cd7004
# Full Hash: 1c3926cd7004e7cba4b2265caee686d6201d21d2
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Move VideoConduit to call worker thread for mCall and related access. r=ng,bwc
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112851
# ==============================================================================

diff -r 844625c1d96d -r 1c3926cd7004 dom/media/webrtc/libwebrtcglue/VideoStreamFactory.cpp
--- a/dom/media/webrtc/libwebrtcglue/VideoStreamFactory.cpp	Sun Apr 18 22:14:28 2021 +0200
+++ b/dom/media/webrtc/libwebrtcglue/VideoStreamFactory.cpp	Wed Apr 07 11:36:28 2021 +0200
@@ -112,12 +112,10 @@
 }
 
 void VideoStreamFactory::SetCodecMode(webrtc::VideoCodecMode aCodecMode) {
-  MOZ_ASSERT(NS_IsMainThread());
   mCodecMode = aCodecMode;
 }
 
 void VideoStreamFactory::SetSendingFramerate(unsigned int aSendingFramerate) {
-  MOZ_ASSERT(NS_IsMainThread());
   mSendingFramerate = aSendingFramerate;
 }
 